### ISSUE TITLE
Spotify API reported error code 503

### ISSUE THREAD

**Issue by Yuki338n** [Rigellute/spotify-tui#1068]
When i execute in the console the command "spt" the gui appears for some seconds and then this error shows up. Or if i ignore it and press esc and try to play a song it appears again. What's this? How can i resolve that?

![Screenshot_20230114_125953](https://user-images.githubusercontent.com/109089108/212470706-e9008878-af74-4f1e-a895-baf0afc797ce.png)

**Comment by ChristophSonnleitner** [comment]
I have the same problem. 
It displays all the infos (playlist, currently playing song) and shows my devices. 
But I can't do anything because of the above mentioned error popping up after trying so play something.

(I can toggle pause/play with the spacebar despite the error-window, resulting in the error-message updating to a 403 error "Player command failed: Restriction violated")

**Comment by meierjan** [comment]
My issue is similar.

I am able to run `spt` and now and then the error is displayed. It doesn't seem to be triggered by anything specific I am doing with the interface. It's just popping showing up (but very often). When I `ESC` it away, I am able to see my library and I can control my `spotifyd`.

- I do have a premium account (family)
- my playback device (`spotifyd`) is active, working and selected 
- the `spotifyd` doesn't have a space in the name

Happens both on an intel and apple silicone macbook with macOS 12.5.1

**Comment by sewnie** [comment]
duplicate of 9000 issues, just use https://github.com/aome510/spotify-player

**Comment by Yuki338n** [comment]
> duplicate of 9000 issues, just use https://github.com/aome510/spotify-player

Yeah i was thinking about using a different spotify api player, this gui is just garbage

**Comment by sewnie** [comment]
I love this Spotify player NGL, it works as a player without spotifyd. I hope more people here can use it than to use deprecated abandonware.
